src/attestation/src/attest.rs

```diff
@@ -91,59 +96,29 @@ pub fn attest_init_heap() -> Option<usize> {
 }
 
 pub fn get_quote(td_report: &[u8]) -> Result<Vec<u8>, Error> {
-    // Create a GetQuote buffer following TDX GHCI format
-    // Header is 20 bytes (version:u64 + status:u64 + in_len:u32 + out_len:u32)
-    let tdreport_length = td_report.len();
-    let header_size = core::mem::size_of::<ServtdTdxQuoteHdr>();
-    let buffer_size = header_size + tdreport_length + TD_QUOTE_SIZE; // Header + TDReport + space for quote
-    let mut buffer = vec![0u8; buffer_size];
-
-    // Fill GetQuote buffer header using the struct
-    let hdr = ServtdTdxQuoteHdr {
-        version: 1,
-        status: 0,
-        in_len: td_report.len() as u32,
-        out_len: TD_QUOTE_SIZE as u32,
-        data: [],
-    };
-
-    let header_size = core::mem::size_of::<ServtdTdxQuoteHdr>();
-    let hdr_bytes =
-        unsafe { core::slice::from_raw_parts(&hdr as *const _ as *const u8, header_size) };
-    buffer[..header_size].copy_from_slice(hdr_bytes);
-
-    // Copy TDREPORT data immediately after header (offset 20)
-    buffer[header_size..header_size + tdreport_length].copy_from_slice(td_report);
-
-    // Call ghci_get_quote
-    let result = ghci_get_quote(buffer.as_mut_ptr() as *mut c_void, buffer.len() as u64);
-    if result != AttestLibError::Success as i32 {
-        return Err(Error::GetQuote);
-    }
-
-    // Read header response
-    let buffer_ptr = buffer.as_ptr() as *const ServtdTdxQuoteHdr;
-    let (status, quote_length) = unsafe {
-        let hdr = &*buffer_ptr;
-        (hdr.status, hdr.out_len as usize)
-    };
-
-    if status != GET_QUOTE_SUCCESS {
-        return Err(Error::GetQuote);
+    #[cfg(feature = "igvm-attest")]
```

Comment:
`igvm-attest` feature gated get_quote() PR should replace commit `3e8b964`, which includes code changes in ghci.rs. Might be easier to revert commit `3e8b964` and `fdab04d`, then merge commit `8f12544`